spyder/plugins/editor/plugin.py

```diff
@@ -1325,7 +1325,8 @@ def update_font(self):
                 comp_widget = finfo.editor.completion_widget
                 kite_call_to_action = finfo.editor.kite_call_to_action
                 comp_widget.setup_appearance(completion_size, font)
-                kite_call_to_action.setFont(font)
+                if kite_call_to_action is not None:
+                    kite_call_to_action.setFont(font)

```

Comment:
I know you're still working on this, but I'd like to let you know that you can remove all references to `kite_call_to_action` here and in all other places it's used.

Given that we don't support Kite anymore, we shouldn't try to keep its support alive in our codebase.

Reply:
Do you mean we should remove all mentions of kite?

Reply:
For now you can remove all references to `kite_call_to_action`. Then we'll see how to remove the rest of Kite support in another PR.